doc/notebooks/basic_example_grass_jupyter.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Try GRASS GIS in Jupyter Notebook with Python and grass.jupyter\n",
+    "\n",
+    "[<img src=\"../../man/grass_logo.png\" alt=\"GRASS GIS\" style=\"width:200px;\"/>](https://grass.osgeo.org/)\n",
+    "\n",
+    "This is a quick introduction to *GRASS GIS* in a *Jupyter Notebook* using the `grass.jupyter` package and the *Python* scripting language. The `grass.jupyter` package shortens the launch of *GRASS GIS* in *Jupyter Notebook* and provides several useful classes for creating, displaying and saving *GRASS GIS* maps. This notebook can be directly compared with `example_notebook.ipynb` to see how the package improves the integration of *GRASS GIS* and *Jupyter Notebooks*.\n",
```

Comment:
The name of the original notebook is wrong and if you write it as [basic_example.ipynb](basic_example.ipynb), it actually opens the other one.

Reply:
Good catch and good idea!